completion/utilities.py

```diff
@@ -0,0 +1,23 @@
+#Python imports
+from __future__ import unicode_literals, absolute_import
+
+#Project imports
+from .exceptions import UnavailableCompletionData
+from .models import BlockCompletion
+from .waffle import visual_progress_enabled
+
+#Library imports
+
+def get_key_to_last_completed_course_block(user, enrollment):
+    if not visual_progress_enabled(enrollment.course_id):
+        raise UnavailableCompletionData
+
+    try:
+        last_completed_block = BlockCompletion.get_latest_block_completed(user, enrollment.course_id)
+        return last_completed_block.block_key
+    except AttributeError, RuntimeError:
```

Comment:
Instead of checking for AttributeError, please just check whether `last_completed_block` is truthy or not